### DISCUSSION TITLE
How to reference field's value by Jinja in child table?

### DISCUSSION THREAD

**Discussion by kittiu** [frappe/print_designer#143] [Q&A] [answered]
For the document itself, we can use, i.e., {{ doc.total_amount }} and will get the value.
But for child table, says, the filed name is amount, I am trying to use {{ amount }}, {{ doc.amount }}, {{ line.amount }} but not success. 

Question is, how can I reference field value using jinja?

Note: I can use dynamic text but it is not what we want. Some time, I might want to reformat it the way we want.

![image](https://github.com/frappe/print_designer/assets/1973598/292fccc1-cb0b-473b-8df7-8b44639a7941)

Thanks!

**Comment by maharshivpatel** [comment]
@kittiu you can reference like `{{ row.amount }}`

**Reply by kittiu** [reply, marked as answer]
Super fast response! Thanks!
